base.Dockerfile

```diff
@@ -9,7 +9,7 @@ ENV DEBIAN_FRONTEND=noninteractive \
     VERSION_GIT=1:2.13.x \
     VERSION_FPM=1.8.x \
     # jfrog doesn't support 1.8.x format yet
-    VERSION_JFROG=1.9.0 \
+    VERSION_JFROG=1.10.3 \
```

Comment:
`1.10.x`?

Reply:
>       # jfrog doesn't support 1.8.x format yet 

:D

Reply:
:+1: It wasn't clear to me that this has is somehow special, I assumed all these are just passed to `apt`.